src/main/java/build/buildfarm/worker/shard/Worker.java

```diff
@@ -606,7 +608,8 @@ public void start() throws ConfigurationException, InterruptedException, IOExcep
     // Create the appropriate writer for the context
     CasWriter writer;
     if (!configs.getWorker().getCapabilities().isCas()) {
-      writer = new RemoteCasWriter(backplane.getWorkers(), workerStubs);
+      Retrier retrier = new Retrier(Backoff.sequential(5), Retrier.DEFAULT_IS_RETRIABLE);
```

Comment:
This is interesting that there is some backoff being introduced here. Seems a bit orthogonal to Blake3, so seemed worth calling out.